### DISCUSSION TITLE
Video

### DISCUSSION THREAD

**Discussion by rmccar** [ONSdigital/design-system#720] [Feedback]
Use this place to discuss the [video component](https://ons-design-system.netlify.com/components/video/) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by trevorsaint** [comment]
An aspect ratio of **16:9** is not accurate when a padding top of `25px` has been added. This throws out the aspect ratio. Removing this property puts the video back to a proper aspect ratio.

I would suggest removing the `25px` padding top.

Current use (incorrect aspect ratio):
<img width="608" alt="Screenshot 2020-11-04 at 11 48 30" src="https://user-images.githubusercontent.com/4989027/98108522-00106680-1e94-11eb-9a90-c00e2cd47d17.png">

Proposed change (correct aspect ratio):
<img width="604" alt="Screenshot 2020-11-04 at 11 48 39" src="https://user-images.githubusercontent.com/4989027/98108558-11597300-1e94-11eb-8f95-c45f7b0f1725.png">

**Comment by jrbarnes9** [comment]
We should ensure support for Vimeo as well as YouTube hosting due to the ONS using Vimeo

**Reply by jrbarnes9** [reply]
Raised #2489

**Comment by jrbarnes9** [comment]
Include option to display transcripts (links or collapsed content) directly beneath videos.

To improve situations like this: 
![Screenshot 2023-02-01 at 12 11 51](https://user-images.githubusercontent.com/43346934/216065142-a5951735-3d9d-4d1c-af47-a21acd5d26fb.png)

**Reply by jrbarnes9** [reply]
Raised #2596
